<ui version="4.0" >
 <class>QFillDatabase</class>
 <widget class="QWidget" name="QFillDatabase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="groupBox2" >
     <property name="title" >
      <string>Documents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="_2" >
        <item>
         <widget class="QLabel" name="TextLabel1" >
          <property name="text" >
           <string>Path to Directory:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="KUDirectory" >
          <property name="minimumSize" >
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_3" >
        <item>
         <widget class="QLabel" name="textLabel1_2" >
          <property name="text" >
           <string>Default mime-type:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DefaultMIMEType" >
          <property name="text" >
           <string>text/xml</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QCheckBox" name="FixLanguage" >
          <property name="text" >
           <string>Fix the language</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Language" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox1" >
     <property name="title" >
      <string>Topics</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QCheckBox" name="Topics" >
        <property name="text" >
         <string>Documents are catogorized on a directory basis?</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_5" >
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Category depth :</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer7" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>251</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KIntSpinBox" name="Depth" >
          <property name="value" >
           <number>1</number>
          </property>
          <property name="maxValue" stdset="0" >
           <number>2</number>
          </property>
          <property name="minValue" stdset="0" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_6" >
        <item>
         <widget class="QCheckBox" name="HasParent" >
          <property name="text" >
           <string>Has parent :</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer8" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="ParentName" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChooseBtn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Choose</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>HasParent</sender>
   <signal>toggled(bool)</signal>
   <receiver>ParentName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>88</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel" >
     <x>248</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HasParent</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChooseBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel" >
     <x>328</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FixLanguage</sender>
   <signal>toggled(bool)</signal>
   <receiver>Language</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>19</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel" >
     <x>213</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topics</sender>
   <signal>toggled(bool)</signal>
   <receiver>HasParent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>188</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel" >
     <x>56</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topics</sender>
   <signal>toggled(bool)</signal>
   <receiver>ParentName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel" >
     <x>235</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topics</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChooseBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>285</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel" >
     <x>288</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Topics</sender>
   <signal>toggled(bool)</signal>
   <receiver>Depth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>223</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel" >
     <x>296</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
